class NotFoundError(Exception):
    pass


class UnauthorizedError(Exception):
    pass
